### PR TITLE
chore: bump version to beta.4 and update changelog

### PR DESCRIPTION
**Pull request by kara**
Updates for beta.4

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,111 @@
+<a name="2.0.0-beta.4"></a>
+# [2.0.0-beta.4 unobtainium-sunglasses](https://github.com/angular/material2/compare/g3_v2_0...2.0.0-beta.4) (2017-05-12)
+
+### Breaking Changes
+
+* The `MdIconModule` no longer exports `HttpModule`, so you'll need to import it if you're using it elsewhere in your app.
+* The `forRoot` method on our modules has been removed. You should import our modules directly without this method.
```
Comment (jelbourn) [inline CHANGELOG.md]:
```md
* `MdIconModule` no longer imports `HttpModule`. If your application depended on `Http` being provided through `MdIconModule`, you should now directly import `HttpModule` into your application.
* The `forRoot` method on all Angular Material modules has been removed. It was previously deprecated and a no-op. Importing the modules directly will have the same effect.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,111 @@
...
+* **chips:** wrong margin on single chip ([#4366](https://github.com/angular/material2/issues/4366)) ([5d03c1a](https://github.com/angular/material2/commit/5d03c1a)), closes [#4359](https://github.com/angular/material2/issues/4359)
+* **compatibility:** error in theme check with angular universal ([#3872](https://github.com/angular/material2/issues/3872)) ([a65d2f4](https://github.com/angular/material2/commit/a65d2f4)), closes [#3870](https://github.com/angular/material2/issues/3870)
+* **compatibility:** throw better error when wrong prefix is used ([#3871](https://github.com/angular/material2/issues/3871)) ([2d50044](https://github.com/angular/material2/commit/2d50044))
+* **connected-overlay:** better handling of dynamic content ([#4250](https://github.com/angular/material2/issues/4250)) ([525ce1e](https://github.com/angular/material2/commit/525ce1e)), closes [#4155](https://github.com/angular/material2/issues/4155)
+* **core:** allow users to disable the sanity checks ([#4178](https://github.com/angular/material2/issues/4178)) ([16bba72](https://github.com/angular/material2/commit/16bba72)), closes [#4125](https://github.com/angular/material2/issues/4125)
```
Comment (jelbourn) [inline CHANGELOG.md]: I'd re categorize this one as a feature

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,111 @@
...
+* **theming:** fix broken sass expressions nested in theme classes ([#4145](https://github.com/angular/material2/issues/4145)) ([355f8b7](https://github.com/angular/material2/commit/355f8b7)), closes [#4077](https://github.com/angular/material2/issues/4077)
+* **tooltip:** wrong position when using OnPush change detection ([#3671](https://github.com/angular/material2/issues/3671)) ([edf01c0](https://github.com/angular/material2/commit/edf01c0)), closes [#3497](https://github.com/angular/material2/issues/3497)
+* **universal:** gate several browser-specific bits on being on the browser ([#4251](https://github.com/angular/material2/issues/4251)) ([f27df86](https://github.com/angular/material2/commit/f27df86))
+
+
+### Features
+
+* **card:** allow md-card-title and subtitle to be used as attributes ([#4122](https://github.com/angular/material2/issues/4122)) ([a200024](https://github.com/angular/material2/commit/a200024))
+* **datepicker:** merge datepicker branch into master ([#4404](https://github.com/angular/material2/issues/4404)) ([123d7ec](https://github.com/angular/material2/commit/123d7ec))
```
Comment (jelbourn) [inline CHANGELOG.md]: Make this the first item and change the message to "add initial datepicker"

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,111 @@
...
+* **list:** option to disable ripples for all items ([#4159](https://github.com/angular/material2/issues/4159)) ([7f0f473](https://github.com/angular/material2/commit/7f0f473)), closes [#4149](https://github.com/angular/material2/issues/4149)
+* **observe-content:** add debounce option and other improvements ([#2404](https://github.com/angular/material2/issues/2404)) ([244aece](https://github.com/angular/material2/commit/244aece))
+* **overlay:** add scroll handling strategies ([#4293](https://github.com/angular/material2/issues/4293)) ([c8ec981](https://github.com/angular/material2/commit/c8ec981))
+* **overlay:** expose interface for custom positions ([#4374](https://github.com/angular/material2/issues/4374)) ([ea8241a](https://github.com/angular/material2/commit/ea8241a))
+* **pixacto:** use TypeScript for tooling and refactor into tools/ ([#3732](https://github.com/angular/material2/issues/3732)) ([25a6881](https://github.com/angular/material2/commit/25a6881))
```
Comment (jelbourn) [inline CHANGELOG.md]: Remove this one (not user-facing)

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,111 @@
...
+* **progress-spinner:** add support for custom stroke-width ([#4113](https://github.com/angular/material2/issues/4113)) ([b846a27](https://github.com/angular/material2/commit/b846a27)), closes [#3934](https://github.com/angular/material2/issues/3934)
+* **screenshot:** add control panel for pixacto ([#4055](https://github.com/angular/material2/issues/4055)) ([b2ca0f5](https://github.com/angular/material2/commit/b2ca0f5))
```
Comment (jelbourn) [inline CHANGELOG.md]: Remove this one

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,111 @@
+<a name="2.0.0-beta.4"></a>
+# [2.0.0-beta.4 unobtainium-sunglasses](https://github.com/angular/material2/compare/g3_v2_0...2.0.0-beta.4) (2017-05-12)
+
+### Breaking Changes
+
+* The `MdIconModule` no longer exports `HttpModule`, so you'll need to import it if you're using it elsewhere in your app.
+* The `forRoot` method on our modules has been removed. You should import our modules directly without this method. 
+
+### Bug Fixes
```
Comment (jelbourn) [inline CHANGELOG.md]: Should add a section above bug fixes that's "Highlights" which mentions the datepicker, ability to turn off sanity checks, and anything else you think is noteworthy.

**Comment by kara** [comment]
Verbal LGTM from @jelbourn

**Pull request merged**